opensearch.yml

```diff
@@ -2,12 +2,12 @@
 
 - name: opensearch installation & configuration
   hosts: os-cluster
-  gather_facts: false
+  gather_facts: true
```

Comment:
Can you explain what does this do?

Reply:
it will collect and create predefined variables such as `ansible_distribution` which is used to identify the OS flavor.

Reply:
Got it.